R/travis-vars.R

```diff
@@ -4,24 +4,29 @@
 #' Functions around public and private variables available in Travis CI builds.
 #'
 #' `travis_get_vars()` calls the "/settings/env_vars" API.
-#'
-#' @inheritParams travis_set_pat
-#'
+#' @template repo
+#' @template quiet
 #' @export
 #' @examples
 #' \dontrun{
 #' # List all variables:
 #' travis_get_vars()
 #' }
-travis_get_vars <- function(repo = github_repo(), token = travis_token(repo)) {
-  req <- TRAVIS_GET3(sprintf("/repo/%s/env_vars", encode_slug(repo)),
-    token = token
-  )
-  httr::stop_for_status(
-    req,
-    sprintf("get environment variables for %s from Travis CI", repo)
+travis_get_vars <- function(repo = github_repo(), endpoint = get_endpoint(),
+                            quiet = FALSE) {
+
+  req <- travis(
+    path = sprintf("/repo/%s/env_vars", encode_slug(repo)),
+    endpoint = endpoint
   )
-  new_travis_env_vars(httr::content(req))
+
+  if (status_code(req$response) == 200) {
```

Comment:
I think we need `stop_for_status()` everywhere?

Reply:
Yes, done.